src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). It's better to style elements using classes to avoid potential conflicts and enhance reusability.